packages/datastore/datastore/src/react/hooks/query.ts

```diff
@@ -1,129 +1,132 @@
-import { useEffect, useReducer, useCallback, Dispatch } from "react";
+import { useEffect, useReducer, useCallback, Dispatch, useMemo } from "react";
```

Comment:
Kinda pushed that improvement by mistake. It is actually good to have it this way.